frontend/src/components/Modals/CoffeeChatDetailsModal.tsx

```diff
@@ -51,12 +53,18 @@ const CoffeeChatModal: React.FC<Props> = ({
     }
     setMembersInCategory(
       category in suggestions
-        ? suggestions[category].filter((chat) =>
-            submittedChats.every((submittedChat) => submittedChat.otherMember.netid !== chat.netid)
+        ? suggestions[category].filter(
+            (chat) =>
+              submittedChats.every(
+                (submittedChat) => submittedChat.otherMember.netid !== chat.netid
+              ) &&
+              approvedArchivedChats.every(
+                (approvedArchivedChat) => approvedArchivedChat.otherMember.netid != chat.netid
```

Comment:
```suggestion
                (approvedArchivedChat) => approvedArchivedChat.otherMember.netid !== chat.netid
```
Just adding this for better strong type checking! 

Also another Suggestion (Feel free to ignore) -> You can also try putting both the submitted/Archived Net ids in a set and filter based on that for constant look checks!

Reply:
sounds good, thank you!